data/content_scripts/posting_process/sendkeys.js

```diff
@@ -0,0 +1,766 @@
+// Cross-broswer implementation of text ranges and selections
+// documentation: http://bililite.com/blog/2011/01/17/cross-browser-text-ranges-and-selections/
+// Version: 2.6
+// Copyright (c) 2013 Daniel Wachsstock
+// MIT license:
```

Comment:
But this won't work. What I previously tried --
Inject a small script always(on every page load). So, this small script will be present in an iframe if any. On receiving the privly url from privly-applications(which is the final step, only posting remains now), inject the library into the `resultTab` and then access the lib object from the small script present in the iframe. Unfortunately, I can't access the lib object from the small script.
I think our question is more or less the same as [this](https://forums.mozilla.org/viewtopic.php?t=11092&p=23524) but I'm unable to find an elegant way to do it :/